docs/source/concept_models/vivarium_csu_multiple_myeloma_2/concept_model.rst

```diff
@@ -147,15 +141,37 @@ set of treatment categories based on treatment guidelines from the `NCCN
 5.0 Simulation Scenarios
 ++++++++++++++++++++++++
 
+Our scenarios differ in how treatment is assigned.
+
+We know that our scenarios will roughly correspond to:
+
+* Baseline scenario: Sophisticated treatment assignment, with postprocessing rules (modifying the prediction model's probabilities) about Isa and Dara by year, and a rule that there is probability 0 of Isa directly following Dara
+* Alternative scenario 1: Naive treatment assignment (not dependent on covariates) with the same postprocessing rules as baseline
+* Alternative scenario 2: Same as baseline, but with postprocessing rules modified such that there is 5% uptake of Isa in second line following Dara first-line treatment by 2028 -- need to define how this replaces other things
+* Alternative scenario 3: Same as baseline, but with postprocessing rules modified such that there is 5% uptake of Isa first-line treatment by 2028 -- need to define how this replaces other things
+
+.. todo::
+
+  Rigorously define these scenarios. In particular, the scenarios above say "Isa" when we will actually want some split between Isa+PI+Dex, Isa+IMID+Dex, etc.
+
 6.0 Simulation Parameters
 +++++++++++++++++++++++++
 
 6.1 Locations
 -------------
 
+United States.
+
 6.2 Population and Randomness
 -----------------------------
 
+Population description:
+
+* Cohort type: Prospective closed cohort of individuals aged 15 years and older (open cohort if we only examine on age 50+ population)
```

Comment:
I think the cohort type could use a bit more clarity - this ended up being a bit of a pain point in the IV iron sim when we had to switch from a closed to open cohort halfway through sim development. Basically, I think just stating the rationale behind each cohort type so it's more clear what to prioritize would be valuable.

Reply:
We are definitely going to use a closed cohort at the Vivarium level. I think that the parenthetical is supposed to describe that it can be _considered_ an open cohort if we only look at results for 50+ (or 30+ for that matter, since the sim duration is 15 years). I will clarify.